### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/ACS/acs_polarization_tools/acs_polarization_tools.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#195]
## What
An accessibility scan flagged 5 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/ACS/acs_polarization_tools/acs_polarization_tools.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell2">` (selector: `#codecell2`)
- `<pre id="codecell6"><span></span>{&apos;description&apos;: &apos;WFC filters use MJD corresponding to 2020-01-01. HRC filters use MJD corresponding to 2007-01-01.&apos;}
</pre>` (selector: `#codecell6`)
- `<pre id="codecell7">` (selector: `#codecell7`)
- `<pre id="codecell8"><span></span>Stokes I = 169652.47890399996, Stokes Q = -8276.254216000001, Stokes U = -4644.582265627395
Fractional polarization = 5.60%, Position Angle = 180.8631543698736 deg
</pre>` (selector: `#codecell8`)
- `<pre id="codecell9">` (selector: `#codecell9`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
